sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/selenium/conditional/access/AADConditionalAccessIT.java

```diff
@@ -0,0 +1,100 @@
+package com.azure.test.aad.selenium.conditional.access;
+
+import com.azure.test.aad.selenium.AADSeleniumITHelper;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.annotation.RegisteredOAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.registration.ClientRegistrationRepository;
+import org.springframework.security.oauth2.client.web.OAuth2AuthorizedClientRepository;
+import org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.Map;
+
+import static com.azure.spring.test.EnvironmentVariable.AAD_CUSTOM_ENDPOINT_CLIENT_ID;
+import static com.azure.test.aad.selenium.AADSeleniumITHelper.createDefaultProperties;
+import static org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction.oauth2AuthorizedClient;
+
+public class AADConditionalAccessIT {
+
+    private AADSeleniumITHelper aadSeleniumITHelper;
+
+    private static final String CUSTOM_LOCAL_FILE_ENDPOINT = "http://localhost:8081/call-custom";
+    private static final Logger LOGGER = LoggerFactory.getLogger(AADConditionalAccessIT.class);
+
+    @Ignore
```

Comment:
So no test case will be run?

Reply:
We discussed and haven't thought of a better way yet, so this case is changed to manual.